OpenRA.Mods.Common/Widgets/ConfirmationDialogs.cs

```diff
@@ -56,6 +56,37 @@ public static void CancelPrompt(string title, string text, Action onCancel = nul
 			};
 		}
 
+		public static void PromptAbortMission(World world, string title, string text, Action onAbort, Action onCancel = null, Action closeMenu = null)
```

Comment:
Do you know why the original PR introduced this duplicate method?  It would be better to generalize `PromptConfirmAction` to cover this use case instead of duplicating code and yaml.